bundles/CoreBundle/Resources/translations/nl.extended.json

```diff
@@ -613,6 +613,9 @@
     "right": "right",
     "provide_split_view": "Provide Split View",
     "allow_multiple_assignments": "Allow Multiple Assignments",
+    "disable_delete": "Verwijderen uitschakelen",
+    "disable_add": "Toevoegen uitschakelen",
+    "disable_create": "Aanmaken uitschakelen",
```

Comment:
Do we need to add it to the other languages as well?

Reply:
Looks like they automatically fall back to English.

Reply:
Translations please only for EN, other languages are managed at POEditor.